.github/workflows/check-3rd-party.sh

```diff
@@ -5,7 +5,7 @@ status=0
 
 for line in `sed -ne 's/[[:space:]-]*uses:[[:space:]]*//p' $1 | sed -e 's/\s*#.*$//'`; do
   author=`echo $line | awk -F/ '{print $1}'`
-  if [[ $author == "actions" || $author == "protocol" ]]; then continue; fi
+  if [[ $author == "." || $author == "actions" || $author == "protocol" ]]; then continue; fi
```

Comment:
document what the "dot" author is? at least I have no idea what that means

Reply:
I added some documentation.